lib/que/sql.rb

```diff
@@ -58,9 +63,9 @@ module Que
 
     :insert_job => %{
       INSERT INTO que_jobs
-      (queue, priority, run_at, job_class, args)
+      (queue, priority, run_at, job_class, retryable, args)
       VALUES
-      (coalesce($1, '')::text, coalesce($2, 100)::smallint, coalesce($3, now())::timestamptz, $4::text, coalesce($5, '[]')::json)
+      (coalesce($1, '')::text, coalesce($2, 100)::smallint, coalesce($3, now())::timestamptz, $4::text, $5::bool, coalesce($6, '[]')::json)
```

Comment:
After discussion, it's not necessary to coalesce as there is a `DEFAULT true` in play.